rutabaga.gemspec

```diff
@@ -17,7 +17,8 @@ Gem::Specification.new do |gem|
   gem.license       = 'MIT'
 
   gem.add_runtime_dependency 'turnip', ['~> 2.1','>= 2.1.1']
-  gem.add_runtime_dependency 'activesupport'
+  gem.add_runtime_dependency 'rspec', '~> 3.4.0'
```

Comment:
Changed to support all version of rspec 3 up to 3.5 in commit https://github.com/simplybusiness/rutabaga/pull/19/commits/c04faf5b4ff0fba8486699b599c6f6dc9b80b77d